manager_janitor.go

```diff
@@ -468,16 +579,29 @@ func filterFeedable(addFeeds [][]*PIndex) (rv [][]*PIndex) {
 	return rv
 }
 
+const (
+	PHASE_CHANGE     = "phaseChange"
```

Comment:
PHASE_CHANGE is very generic. And the same for HIBERNATE_PINDEX too since we have multiple levels of hiberation status. 
Better to have names indicative of what it does?

HIBERNATION_OP_WARM_COLD / OP_WARM_COLD / REQ_WARM_COLD etc ..something better than these

Reply:
HIBERNATE_PINDEX is when an index is changing to cold i.e. being hibernated.
PHASE_CHANGE is generic, agreed. So I think PHASE_CHANGE could be replaced by ACTIVATE_PINDEX?
Having a separate var for each type of transition will make it very verbose I think? Since the changes when an index changes to hot or warm are the same, I thought keeping a common var name should suffice.
Just my thoughts.